spec/spec_helper.rb

```diff
@@ -1,2 +1,8 @@
 $LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
 require 'trix'
+require 'rails/version'
+require 'action_dispatch'
+require 'action_controller/metal/live'
+require 'action_controller/test_case'
```

Comment:
Ok - that's fine. Can you just go through and make sure all of the things you have required are fully necessary to be required? Not sure why we need `require 'action_controller/metal/live'`